src/inet/UDPEndPoint.cpp

```diff
@@ -1419,7 +1420,7 @@ CHIP_ERROR UDPEndPointImplSockets::IPv6JoinLeaveMulticastGroupImpl(InterfaceId a
     }
 #endif // CHIP_SYSTEM_CONFIG_USE_PLATFORM_MULTICAST_API
 
-#if defined(INET_IPV6_ADD_MEMBERSHIP) && defined(INET_IPV6_DROP_MEMBERSHIP)
+#ifdef IPV6_MULTICAST_IMPLEMENTED
```

Comment:
Who's in charge of setting/removing this define? Wouldn't it be easier to just make this code unconditional? Otherwise it's one more area where we may have divergence between platform and product in terms of a configuration variable between a common library versus build-from-full-source.

All IPv6 stacks that could realistically be used by Matter support multicast.

Reply:
This define will be remove shortly, for now it's simply to allow all CI job to pass without failure